File: Semester 5 Fall 2024/CL3001-Computer Networks - Lab/Lab Manuals/Lab Manual # 1/C code/Q1PartB.c
#include <stdio.h>

int main(int argc,char * argv[])
{
	if(argc!=2)
	{
		printf("Invalid Parmaters! Give one file name.\n");
		return 0;
	}

	FILE *read = fopen(argv[1], "r");
	if(read==NULL)
	{
		printf("Cannot open input file!\n");
		return 0;
	}

	FILE *write = fopen("PartB_Output.txt", "w"); 
	if(write==NULL)
	{
		printf("Cannot open output file!\n");
		return 0;
	}

	int data=fgetc(read);
	char ch;

	while(data!=EOF)
	{	
		if (data >= 48 && data <= 57)
		{
			ch = (char)data;
			fputc(ch, write);
			fputc(' ', write);
		}
		data=fgetc(read);
	}

	printf("Integer File generated successfully! with the name: 'PartB_Output.txt'\n");
	
	fclose(read);
	fclose(write);
	return 0;
}

File: Semester 5 Fall 2024/CL3001-Computer Networks - Lab/Lab Manuals/Lab Manual # 1/Python code/Q1PartB.py
def main():
    outputFile=open("PartB_Output.txt",'w')
    
    with open("Input File.txt", 'r') as f:
        for line in f:
            
            words=line.split()
            
            for word in words:
                
                printSpace=False
                for ch in word:
                    if(ch>='0' and ch<='9'):
                        outputFile.write(ch)
                        printSpace=True
                
                if(printSpace==True):
                    outputFile.write(" ")
            outputFile.write("\n")
    outputFile.close()
                
        
    
    




main()
